specification.md

```diff
@@ -251,8 +251,8 @@ The workflow data output is the data output of the last executed workflow state.
 
 ```yaml
 stateDataFilter:
-  input: "${ .orders }"
-  output: "${ .provisionedOrders }"
+  input: "${.orders}"
```

Comment:
that was just coincidental, because we discussed different markups while in the making. I changed it in bulk so it got rid of the spaces. They're okay to keep if you prefer. I'll have to redo all changes on the latest main anyways and will keep the spaces.